terraform/modules/tf_globals/alerts_firehose/main.tf

```diff
@@ -24,3 +28,12 @@ resource "aws_kinesis_firehose_delivery_stream" "streamalerts" {
   }
 }
 
+// CloudWatch Log Group: Firehose
+resource "aws_cloudwatch_log_group" "firehose" {
+  name              = "/aws/kinesisfirehose/${local.stream_name}"
+  retention_in_days = var.cloudwatch_log_retention
+
+  tags = {
+    Name = "StreamAlert"
```

Comment:
should this be `Application` ?

Reply:
we standardized on `Name` here (for whatever reason) so updating here means updating 100+ other places